table-expressions.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Table Expressions
+summary: Table expressions define a data source in SELECT clauses and INSERT statements.
+toc: false
+---
+
+Table expressions define a data source in SELECT clauses and INSERT statements.
```

Comment:
Similar to my feedback for `sql-expressions.md`, I think the auto-generated toc is sufficient for listing the page contents. So let's remove `Introduction` and use something like the following before the toc:

```
Table expressions define a data source in [`SELECT`](select.html) clauses and [`INSERT`](insert.html) statements. This page provides details on the possible types of table expressions.`
```